go.mod

```diff
@@ -7,10 +7,10 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/levigross/exp-html v0.0.0-20120902181939-8df60c69a8f5
 	github.com/mattermost/mattermost-plugin-api v0.0.26-0.20220223141232-cb8b1984774a
-	github.com/mattermost/mattermost-server/v6 v6.5.0
+	github.com/mattermost/mattermost-server/v6 v6.7.2
```

Comment:
What was the reason for this change?

Reply:
took the pull from the latest master from matter most.This came from there